be/src/vec/exec/scan/new_olap_scan_node.cpp

```diff
@@ -205,7 +205,9 @@ Status NewOlapScanNode::_build_key_ranges_and_filters() {
         _runtime_profile->add_info_string("PushDownPredicates",
                                           olap_filters_to_string(_olap_filters));
         _runtime_profile->add_info_string("KeyRanges", _scan_keys.debug_string());
-        _runtime_profile->add_info_string("TabletIds", tablets_id_to_string(_scan_ranges));
+        if (!_scan_ranges.empty()) {
```

Comment:
better to do this check inside `tablets_id_to_string()`?
And if `_scan_ranges` is empty, it can return `empty` or `N/A`?